### PR TITLE
Worker tests

### PR DESCRIPTION
**Pull request by vogdb**

### CODE REVIEW

**Comment by vogdb** [comment]
@tomaszdurka please review. It's a very short and pleasant PR ;-)

File: lib/worker.js [hunk 1]
```diff
@@ -36,9 +36,14 @@ module.exports = (function() {
 
     sockjsServer = sockjs.createServer(sockjsOptions);
     this._connectionTimeouts = {};
+    this._httpServer = null;
     listen.call(this, port, sslOptions);
   }
 
+  Worker.prototype.stop = function() {
+    this._httpServer.close();
```
Comment (tomaszdurka) [inline lib/worker.js]: Shall we check if `httpServer` is there?

**Comment by vogdb** [comment]
@tomaszdurka please look

**Comment by vogdb** [comment]
@tomaszdurka please review

**Comment by vogdb** [comment]
mr. Tomasz!

**Comment by tomaszdurka** [comment]
lgtm

**Pull request merged**